sky/clouds/azure.py

```diff
@@ -377,6 +377,12 @@ def _failover_disk_tier() -> Optional[resources_utils.DiskTier]:
             'resource_group': f'{cluster_name.name_on_cloud}-{region_name}',
         }
 
+        disk_performance_tier = Azure._get_disk_performance_tier(r.disk_tier)
```

Comment:
Do we really need that new function or can we have the logic inline, since it looks a bit shallow.

Reply:
Good point! Updated. Thanks!